source/changelog.txt

```diff
@@ -12,6 +12,37 @@
    :depth: 1
    :class: singlecol
 
+v1.0.6
+------
+
+*Released September 14, 2021*
+
+New features in this release:
+
+- You can now run :method:`sh.status()` when connected to a ``mongos``.
```

Comment:
@jeff-allen-mongo This is missing the important word “not”. :)

Fwiw, this feature is TSE-facing. We don’t even document that `sh.status()` takes a second argument (which targets the same use case), so I think we should probably also omit this.

Reply:
Oh no! I read your announcement on Slack too quickly. Thanks for spotting this.